src/envo/envo-edit.owl

```diff
@@ -39742,6 +39744,37 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105014> (biodegradation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1351%2FPAC-REC-10-12-04>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105014> "A biological process during which organic matter is broken down by microorganisms."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105014> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105014> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105014> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105014> "2021-05-06T12:15:46Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "Biodegradation"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "Biologischer Abbauprozess"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradación"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradazione"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105014> "生物分解"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105014> "envoPlastics"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105014> "biodegradation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105014> <http://purl.obolibrary.org/obo/GO_0008150>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105015> (biodegradation of plastic)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1016/j.marpolbul.2011.05.030>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105015> "A biodegradation process during which the carbon in plastic is converted into carbon dioxide."@en)
```

Comment:
This is quite interesting: many "biodegradable plastics" are not degradable to CO2 - this is a great case for resolving some semantic murk - does the GESAMP report or any UN working group glossary say anything on this?

Reply:
@pbuttigieg, I looked up the GESAMP report again and they didn't specify that. I corrently refine the definition as following:

> A biodegradation process during which the molecular weight of plastic is reduced.

based on this [paper](https://doi.org/10.1016/j.matpr.2020.06.096):

> During biodegrada- tion of plastic colonization by the microorganisms on the plastic surface lead to reduce molecular weight of plastics and then first the polymer is converted to its monomers, then the monomers breaks down to carbon dioxide, water and methane termed as min- eralization 

Do you think this will fit better?